tests/test_benchmarks.py

```diff
@@ -0,0 +1,31 @@
+import os
+
+from .utils import *
+
```

Comment:
At column 1: (F403) 'from .utils import *' used; unable to detect undefined names